api/src/main/resources/workbench-api.yaml

```diff
@@ -5345,6 +5385,13 @@ definitions:
         format: int64
         description: 'Timestamp when the user first changed from Unregistered to Registered
           data access level. May be null if the user has never done so.'
+      ras_identity_type:
+        type: string
+        description: The user's identity type the use to login, e.g. eRA Commons, Login.gov
+      ras_identity_name:
```

Comment:
Would we still be populating the `era_commons_linked_username` after RAS? Or no? If we are, then I don't know if we need these fields on the client side or not

Reply:
Most recent docs covers that:
https://docs.google.com/document/d/1sAZq92tdu2QDsf3-xG_XOJ-1EcC2CyET-4_R6WFiTN8/edit?resourcekey=0-f-JzUmaBVbsSy2TWSBPDcQ#heading=h.35h90q375q7a

Right now I added login.gov name, and completion time. Will add more in the future.